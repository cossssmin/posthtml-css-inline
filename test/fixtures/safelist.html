<!DOCTYPE html>
<html>
<head>
  <style>
    .flex {
      display: flex;
    }

    body {
      color: blue;
    }

    h1 {
      text-transform: uppercase;
    }

    p {
      color: red;
    }
  </style>
</head>
<body>
  <h1>Heading</h1>
  <p class="flex">Content</p>
</body>
</html>
