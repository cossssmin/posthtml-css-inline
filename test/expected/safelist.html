<!DOCTYPE html>
<html>
<head>
  <style>
    .flex {
      display: flex;
    }

    body {
      color: blue;
    }

    h1 {
      text-transform: uppercase;
    }

    p {
      color: red;
    }
  </style>
</head>
<body>
  <h1 style="text-transform: uppercase">Heading</h1>
  <p class="flex" style="color: red">Content</p>
</body>
</html>
